cookietemple/lint/template_linter.py

```diff
@@ -663,32 +676,34 @@ def check_section(self, section_items, section_name: str, main_linter: TemplateL
         #
         #   2.) a number, stating the minimum number of items in a section (-1 if number does not care)
         check_set = {
-            'bumpversion': [('current_version', '*'), 1],
-            'bumpversion_files_whitelisted': [('dot_cookietemple', '.cookietemple.yml'), -1],
-            'sync_level': [('ct_sync_level', 'patch|minor|major'), 1],
-            'sync_files_blacklisted': [('changelog', 'CHANGELOG.rst'), -1]
+            'bumpversion': [[('current_version', '*')], 1],
```

Comment:
Actually all of these look more like tuples(tuple, int). 
Maybe they should not be lists?